### PR TITLE
Stabilize ExampleSign anchors, add search terms

### PR DESCRIPTION
**Pull request by xPanini**
# tl;dr

- Add a comment containing "ExampleSign" and "ExampleCommandCaller" so it can be searched by either term
- Adjust sign anchors so they're more stable

### Description of the Change
This PR are subtle changes to the ExampleCommandSign aimed to improve placement stability and make it easier to search for in the GitHub search engine. 

A comment containing "ExampleSign" and "ExampleCommandCaller" has been added so that the ExampleMod can be searched by either term. ExampleCommandSign does not encompass either term and can not be easily searched for without knowing of its existence.

The anchors have been adjusted to be able to place the ExampleSign on trees, make the sign less jumpy to place, and to ensure all tiles are anchored throughout the tiles. 

### Alternate designs
N/A

### Why this should be merged into tModLoader
With these changes, the ExampleCommandSign example will be a much stronger example of how to work with signs in tModLoader. It will also help increase education with the ability to search multiple terms for it. 

### Benefits
A more performant example
Increased searchability in the GitHub search engine

### Possible drawbacks
No drawbacks can be determined from this PR

### Applicable Issues
N/A

### Sample Usage
See [https://github.com/tModLoader/tModLoader/commit/c31e56db7042b68c7efd0182bd60bccdacaef4ac](https://github.com/tModLoader/tModLoader/commit/c31e56db7042b68c7efd0182bd60bccdacaef4ac)

### CODE REVIEW

**Comment by Jofairden** [comment]
I feel like, you're missing a lot?
The only thing I see is that you've updated the anchors

**Comment by xPanini** [comment]
Right! The comment at the top was added too. Very small commit to ensure the ExampleMod is working to its fullest.

**Comment by Jofairden** [comment]
Alright, my bad. Your detailed report and link to #608 made me think you changed the signs to search through github with the sign's content or something.

**Comment by xPanini** [comment]
I suppose the detail was a bit excessive, haha! Thanks 👍
